inc/saiswitch.h

```diff
@@ -971,29 +971,32 @@ typedef enum _sai_switch_attr_t
      * @brief The hash object for IPv4 packets going through ECMP
      *
      * @type sai_object_id_t
-     * @flags READ_ONLY
+     * @flags CREATE_AND_SET
      * @objects SAI_OBJECT_TYPE_HASH
-     * @default internal
+     * @allownull true
+     * @default SAI_NULL_OBJECT_ID
```

Comment:
It might make sense to keep the default as internal for these objects, if the platform has specific out of the box settings for these traffic (although it's not defined what they should be in the spec)

However, the sanity check fails, with :
ASSERT FAILED (on line 998): SAI_SWITCH_ATTR_ECMP_HASH_IPV4: default internal currently can be set only on read only objects

What do you suggest ?